src/schemas/Clinical/index.ts

```diff
@@ -459,20 +553,33 @@ const resolvers = {
 
       return convertRegistrationDataToGql(shortName, response);
     },
-    clearClinicalRegistration: async (obj, args, context, info) => {
+    clearClinicalRegistration: async (
+      obj: unknown,
+      args: {
+        shortName: string;
+        registrationId: string;
+      },
+      context: GlobalGqlContext,
+    ) => {
       const { Authorization } = context;
       const { shortName, registrationId } = args;
       const response = await clinicalService.clearRegistrationData(
         shortName,
         registrationId,
         Authorization,
       );
+      //@ts-ignore consult Jaser on modifying clearRegistrationData or remove this check altogether
       if (response.error) {
+        //@ts-ignore
         throw new UserInputError(response.message);
       }
       return true;
     },
```

Comment:
@jaserud I don't think this block is ever called because the response isn't returned by `clinicalService.clearRegistrationData`. Is this correct?